eitprocessing/datahandling/loading/draeger.py

```diff
@@ -85,36 +85,86 @@ def load_from_single_path(
                 events,
                 phases,
                 previous_marker,
-                first_frame,
```

Comment:
Yes. Events and phases (which are now removed from EITData itself) used to have the frame number (index) as an attribute. Since `index` in this function is the frame number _since the first frame_, the actual frame number in the event or phase had to be `index + first_frame`.